tests/test_game.py

```diff
@@ -48,6 +48,16 @@ def test_choose_not_called_if_isValid_raises_exception(self):
         is_valid_mock = Mock(name="is_valid_mock", side_effect=InvalidBet)
         choose_mock = Mock(name="choose_mock")
         with patch("roulette.Table.isValid", is_valid_mock):
-            with self.assertRaises(InvalidBet):
-                self.game.cycle(self.passenger)
+            with patch("roulette.Wheel.choose", choose_mock):
+                with self.assertRaises(InvalidBet):
+                    self.game.cycle(self.passenger)
         choose_mock.assert_not_called()
+
+    def test_cycle_not_executed_if_no_active_players(self):
```

Comment:
better rename to `test_bets_not_placed_for_inactive_player` as cycle is being executed but bets are not placed when player is not playing

Reply:
It is obvious that `placeBets` will not be called if player is not playing. Adding test against it is not much helpful. so can be removed.

Reply:
Done.